bin/index.js

```diff
@@ -435,7 +430,8 @@ async function fromCmd(packageName, opts) {
     const { depth: maxDepth = 4, output } = opts;
     let manifest = null;
 
-    await checkHydrateDB();
+    await setVulnerabilityStrategy();
```

Comment:
Just a question: Why call this func another time ? We want to reset or something ?

Reply:
What do you mean by calling it another time? The fact that its called from "fromCmd" and also "cwdCmd" ? For now, it is due to the fact that depending on CLI options, you could set different strategies.  At the beginning, I was setting a strategy by default by @fraxken suggested me to let the "set" being called only when needed https://github.com/ES-Community/nsecure/pull/75#discussion_r591780389. Let me know if I answered your question